scan.py

```diff
@@ -4,13 +4,22 @@
 
 def show_aggie_pride():
     slogan_list = ['Aggie Pride - Worldwide',
+ adding-slogan
+                    'Aggie Pride!',
+                    'And That\'s On My 1891',
+                    'Aggies ❤️ engineering',
+                    "Aggies code FR",
+		                'Aggies Do',
+                    'Aggies Do!',
+                    'AGGIES VS EVERYBODY',
                    'Aggie Pride!',
                    'And That\'s On My 1891',
                    'Aggies  engineering',
                    "Aggies code FR",
                    'Aggies Do',
                    'Aggies Do!',
                    'GO AGGIES']
+ main
```

Comment:
It looks like you left the word main in after you resolved merge conflicts which is causing the unit-test to fail.  You will need to correct this